pom.xml

```diff
@@ -51,6 +46,10 @@
             <groupId>org.postgresql</groupId>
             <artifactId>postgresql</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-thymeleaf</artifactId>
```

Comment:
Why is this here?

Reply:
need thymeleaf for the index.html (default view)